build.sbt

```diff
@@ -5,6 +5,10 @@ name := "sbt-license-report"
 sbtPlugin := true
 
 publishMavenStyle := false
+bintrayOrganization := Some("sbt")
+name in bintray := "sbt-license-report"
+bintrayRepository := "sbt-plugin-releases"
+
```

Comment:
We don't care about `bintrayReleaseOnPublish` for this one, but I'll fix the license info.